portfolio.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
+    <link rel="stylesheet" href="styles/normalize.css">
+    <link rel="stylesheet" href="styles/styles.css">
+    <title> Portfolio </title>
+  </head>
+
+<body>
+  <header>
+    <h1> Header of website goes here </h1>
+
+    <nav>
+      <ul>
+        <li><a href="index.html"> home </a></li>
+        <li><a href="hobby_blog.html"> blog </a></li>
+        <li><a href="about.html"> about </a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <section class="portfolio">
+  <h2> Portfolio </h2>
+
+    <nav>
+      <li><a href="https://github.com/mcmar7/Calculator.git"><img class="circular--landscape" src="assets/place_holder2.jpg"/></a></li>
+      <li><a href="https://github.com/mcmar7/BankAccounts.git"><img class="circular--landscape" src="assets/place_holder3.jpg"/></a></li>
+      <li><a href="https://github.com/mcmar7/Solar-System.git"><img class="circular--landscape" src="assets/place_holder4.jpg"/></a></li>
+    </nav>
+
+    <ul>
```

Comment:
How else could we group the images and the names to make it easier for positioning?

Coupling them in divs could be one approach! This way you could also place the text over images or something fancy like hover 🎉